handlers/alarms-handler/src/index.ts

```diff
@@ -109,21 +135,10 @@ const handleSnsPublishMessage = async ({
 	}
 
 	const app = messageAttributes.app?.Value;
-	const teams = getTeams(app);
-
-	await Promise.all(
-		teams.map((team) => {
-			const webhookUrl = getTeamWebhookUrl(team);
 
-			const text = message;
+	const text = message;
 
-			console.log(`SNS publish message from ${app} owned by ${team}`);
+	console.log(`SNS publish message from ${app}`);
 
-			return fetch(webhookUrl, {
-				method: 'POST',
-				headers: { 'Content-Type': 'application/json' },
-				body: JSON.stringify({ text }),
-			});
-		}),
-	);
+	return { app, text };
```

Comment:
Totally subjective, and very minor, but rather than assigning to a new const I'd probably just map from one name to the other in the return:

```suggestion
	return { app, text: message };
```